libraries/chain/include/eosio/chain/hotstuff.hpp

```diff
@@ -40,8 +40,6 @@ namespace eosio { namespace chain {
       fc::sha256                          proposal_id = NULL_PROPOSAL_ID; //vote on proposal
       name                                finalizer;
       fc::crypto::blslib::bls_signature   sig;
-
-      hs_vote_message() = default;
```

Comment:
Should we remove the other defaults
- extended_schedule() = default;
- quorum_certificate() = default;
- hs_proposal_message() = default;
- hs_new_view_message() = default;
- hs_new_block_message() = default;
- hs_proposal_message() = default;
- finalizer_state() = default;

Reply:
removed